sumtool/storage.py

```diff
@@ -1,30 +1,61 @@
 import json
-import hashlib
 import os
 from typing import Dict, Optional, Any
+import unicodedata
+import re
 
 STORAGE_DIR = "./data"
 
 
-def dataset_dir(dataset):
+def dataset_dir(dataset: str):
     return f"{STORAGE_DIR}/{dataset}"
 
 
+def model_path(dataset: str, model: str, ext):
+    return f"{dataset_dir(dataset)}/{slugify(model)}-{ext}"
+
+
+def slugify(value, allow_unicode=False):
```

Comment:
minor: seems like your adding types to the other functions, want to add them here?